Snakefile

```diff
@@ -337,6 +357,9 @@ rule foldseek_clustering:
         resultsdir = output_dir / clusteringresults_dir
     conda:
         "envs/foldseek.yml"
+    resources:
+        mem_mb=32 * 1000
+    threads: 16
```

Comment:
This is probably very over-resourced, as average usage seemed to be about 3 vCPUs with very little memory usage.

I avoided honing this down, because it looked like there were already some solid benchmarking runs. Snakemake's default resource requests are 1 vCPU and 1 GB of RAM per task; it did fail at that level.

Reply:
I think this fine for now. In the past, I've done something like:
```
resources:
    mem_mb=lambda wildcards, attempt: attempt *2000,
```

along with the command line param `--retries 3`. It will launch with 2000 the first time, and if it fails will relaunch with 4000, etc.